### PR TITLE
Use https for badges

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/Microsoft/TypeScript.svg?branch=master)](https://travis-ci.org/Microsoft/TypeScript)
-[![npm version](https://badge.fury.io/js/typescript.svg)](http://badge.fury.io/js/typescript)
-[![Downloads](http://img.shields.io/npm/dm/TypeScript.svg)](https://npmjs.org/package/typescript)
+[![npm version](https://badge.fury.io/js/typescript.svg)](https://www.npmjs.com/package/typescript)
```
Comment (mhegazy) [inline README.md]: `https://npmjs.org/package/typescript`

Comment (bootstraponline) [inline README.md]: `https://npmjs.org/package/typescript` redirects to `https://www.npmjs.com/package/typescript` should I still update it?

Comment (mhegazy) [inline README.md]: some entries has one and some has the other. i do not care which one it is as long as they are all the same.

Comment (bootstraponline) [inline README.md]: ok, fixed

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**